src/sysusers/sysusers.c

```diff
@@ -1550,9 +1553,22 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
 
                                 path_kill_slashes(i->uid_path);
                         } else {
+                                _cleanup_free_ char *uid = NULL, *gid = NULL;
+                                if (split_pair(resolved_id, ":", &uid, &gid) == 0) {
+                                        r = parse_gid(gid, &i->gid);
+                                        if (r < 0) {
+                                                log_error_errno(r, "Failed to parse GID: '%s': %m", id);
+                                                return -EBADMSG;
```

Comment:
I think it is not necessary to return `-EBADMSG` here. Please use `return log_error_errno(...);`.

Reply:
Thank you! Cleaner indeed.